Adyen/ApiSerialization/Converter/SaleToPoiMessageConverter.cs

```diff
@@ -8,8 +8,6 @@ internal class SaleToPoiMessageConverter : JsonConverter
     {
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            serializer.NullValueHandling = NullValueHandling.Ignore;
```

Comment:
To be honest, I don't understand how this can help to fix the issue. `WriteJson` is abstaract method of `Newtonsoft.Json.JsonConverter` class which is called by `Newtonsoft.Json` internally when we call `JsonConvert.SerializeObject`. If we create new method with different name it won't be called by `Newtonsoft.Json`.
Also, I don't see how this can break existing integrations. These two converters (`SaleToPoiMessageConverter` and `SaleToPoiMessageSecuredConverter`) are internal and they are used only in `JsonConvertSerializerWrapper` where values for `NullValueHandling` and `MissingMemberHandling` are explicitly provided when `JsonSerializerSettings` is created.

Reply:
Ah, you're right. I didn't see that these functions are inherited from ` : JsonConverter` - Ignore my proposed solution.

As for the breaking part, I initially noticed that the proposed change would change the `NullValueHandling` from `Ignore` (default: `Include`), this will then change the behavior when either the `SaleToPoiMessageConverter.WriteJson(...)`/`SaleToPoiMessageSecuredConverter.WriteJson(...)`-functions are called.

However, I've overlooked that you've moved these fields to the `GetSerializerSettings(...)`-function.

All is good! 👍